### PR TITLE
Wye

### PR DESCRIPTION
**Pull request by wuqianye**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/ListEntriesServlet.java [hunk 1]
```diff
@@ -17,14 +17,10 @@
...
 import com.google.cloud.datastore.Datastore;
 import com.google.cloud.datastore.DatastoreOptions;
-import com.google.cloud.datastore.StructuredQuery.CompositeFilter;
 import com.google.cloud.datastore.Entity;
 import com.google.cloud.datastore.Query;
-import com.google.cloud.datastore.StructuredQuery;
 import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
 import com.google.cloud.datastore.QueryResults;
-import com.google.cloud.datastore.Value;
-import com.google.cloud.datastore.StructuredQuery.OrderBy;
```
Comment (cam-gav) [inline src/main/java/com/google/sps/servlets/ListEntriesServlet.java:27]: why are these imports deleted?

File: src/main/java/com/google/sps/OAuthUtils.java [hunk 2]
```diff
@@ -2,10 +2,8 @@
 
 import com.google.api.client.auth.oauth2.Credential;
 import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
-import com.google.api.client.http.GenericUrl;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.http.javanet.NetHttpTransport;
-import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
```
Comment (cam-gav) [inline src/main/java/com/google/sps/OAuthUtils.java:7]: why are these imports deleted?

File: src/main/java/com/google/sps/servlets/LoginCallbackServlet.java [hunk 3]
```diff
@@ -7,7 +7,6 @@
 import com.google.api.client.http.GenericUrl;
 import com.google.sps.OAuthUtils;
 import java.io.IOException;
-import javax.servlet.ServletException;
```
Comment (cam-gav) [inline src/main/java/com/google/sps/servlets/LoginCallbackServlet.java:10]: why is this import deleted?

File: src/main/java/com/google/sps/servlets/LoginScreenServlet.java [hunk 4]
```diff
@@ -1,10 +1,7 @@
 package com.google.sps.servlets;
 
-import com.google.api.client.auth.oauth2.Credential;
-import com.google.api.services.oauth2.model.Userinfo;
 import com.google.sps.OAuthUtils;
 import java.io.IOException;
-import javax.servlet.ServletException;
```
Comment (cam-gav) [inline src/main/java/com/google/sps/servlets/LoginScreenServlet.java:7]: why are these imports deleted?

File: src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 5]
```diff
@@ -5,10 +5,8 @@
 import com.google.api.client.http.GenericUrl;
 import com.google.sps.OAuthUtils;
 import java.io.IOException;
-import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
```
Comment (cam-gav) [inline src/main/java/com/google/sps/servlets/LoginServlet.java:11]: why are these imports deleted?

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -1,12 +1,54 @@
...
+            </div>
+            <!-- Journal Display Section -->
+            <div class="journal-display">
+                <div class="w-75 h-100 pt-5 d-flex flex-column">
+                    <input class="font-weight-bold w-100 p-2 border-light rounded-lg" type="text" name="" id="entryTitle" placeholder="Title" readonly>
+                    <textarea name="journal-content" class="p-4 rounded-lg flex-grow-1" id="entryContent" readonly></textarea>
+                </div>
+            </div>
+        </main>
+    </body>
```
Comment (cam-gav) [inline src/main/webapp/index.html:55]: looks good

**Review by cam-gav — CHANGES_REQUESTED** [review]

File: src/main/webapp/script.js [hunk 7]
```diff
@@ -1,13 +1,13 @@
 // Copyright 2020 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
```
Comment (jonathantanmy) [inline src/main/webapp/script.js]: Don't change the license text. Same comment on line 10.

File: src/main/webapp/index.html [hunk 8]
```diff
@@ -1,12 +1,54 @@
...
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <title>Journally</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+
+        <!-- Bootstrap -->
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
```
Comment (jonathantanmy) [inline src/main/webapp/index.html:10]: If I remember correctly, Camilo added a version of Bootstrap which also came with a JS file. Can you make sure that it is the same one?

File: src/main/java/com/google/sps/servlets/ListEntriesServlet.java (hunk 1 above)
Comment (wuqianye) [reply src/main/java/com/google/sps/servlets/ListEntriesServlet.java:27]: these imports are never used

File: src/main/webapp/script.js (hunk 7 above)
Comment (wuqianye) [reply src/main/webapp/script.js]: sorry for that, I was change " to ' in the code to keep them consistent and I just did ctrl+h

File: src/main/java/com/google/sps/OAuthUtils.java (hunk 2 above)
Comment (wuqianye) [reply src/main/java/com/google/sps/OAuthUtils.java:7]: imports never used

File: src/main/java/com/google/sps/servlets/LoginCallbackServlet.java (hunk 3 above)
Comment (wuqianye) [reply src/main/java/com/google/sps/servlets/LoginCallbackServlet.java:10]: import never used

File: src/main/java/com/google/sps/servlets/LoginScreenServlet.java (hunk 4 above)
Comment (wuqianye) [reply src/main/java/com/google/sps/servlets/LoginScreenServlet.java:7]: imports never used

File: src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 5 above)
Comment (wuqianye) [reply src/main/java/com/google/sps/servlets/LoginServlet.java:11]: imports never used

File: src/main/webapp/index.html (hunk 8 above)
Comment (wuqianye) [reply src/main/webapp/index.html:10]: same version

**Review by cam-gav — APPROVED** [review]

**Review by jonathantanmy — APPROVED** [review]
